modules/cnv_adding_virtio_drivers_vm_yaml.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv_users_guide.adoc
+
+[[cnv-adding-virtio-drivers-vm-yaml]]
+= Adding `virtio` drivers container disk to a virtual machine
+
+{ProductName} distributes `virtio` drivers for Microsoft Windows as a container disk, available from the Red Hat Container Catalog. To xref:cnv-installing-virtio-drivers-installing-windows[install] or xref:cnv-installing-virtio-drivers-existing-windows[add] these drivers to a Windows virtual machine, attach the `virtio-win` container disk to the virtual machine as a SATA CD drive. 
+
+.Procedure
+
+* Add the `cnv-tech-preview/virtio-win` container disk as a `cdrom` disk in the Windows virtual machine configuration file. The container disk will be downloaded from the registry if it is not already present in the cluster. 
+
+[source,yaml]
+----
+spec:
+  domain:
+    devices:
+      disks:
+        - name: virtioContainerDisk
+          bootOrder: 2 <1>
+          cdrom:
+            bus:sata
+volumes:
+  - containerDisk:
+      image: cnv-tech-preview/virtio-win
+    name: virtioContainerDisk
+----
+<1> {ProductName} boots virtual machines disks in the order defined in the `VirtualMachine` configuration file. You can either define other disks for the virtual machine before the `virtio-win` container disk, or use the optional `bootOrder` parameter to ensure the virtual machine boots from the correct disk.
```

Comment:
The trick is you either specify bootorder for all disks or for none. I didn't have bootorder for other disks but cd-rom only, as it's in yaml example here, and my VM failed to start with "No bootable disk"

Reply:
Interesting! Updated.